NEWS

```diff
@@ -1,3 +1,100 @@
+WiredTiger release 2.6.1, 2015-05-13
+------------------------------------
+
+The WiredTiger 2.6.1 release contains new features, minor API changes and bug
+fixes.
+
+New features:
+
+* Add the ability to disable write ahead logging at a per-table granularity.
+  Accessed via log=(enabled) configuration for WT_SESSION::create API.
+  Partial logging has serious implications for recovery, it should be used
+  with caution.
+  refs #1989
+
+* Add the ability to asynchronously flush transactions via a new
+  WT_SESSION::transaction_sync API.
+  refs WT-1908, #1943
```

Comment:
This API doesn't flush the transactions, it waits for any previous asynchronous transaction flushes to complete.